test/adapter/json_api/links_test.rb

```diff
@@ -13,6 +13,10 @@ class LinkAuthorSerializer < ActiveModel::Serializer
             end
 
             link :other, '//example.com/resource'
```

Comment:
Off topic: given that [JSON API is undecided on relative links](https://github.com/json-api/json-api/issues/898), I'd rather not fill our examples with them